docs/getting_started.rst

```diff
@@ -45,6 +45,11 @@ Note that running the tests is not necessary if you just want to use pydrad.
 Additional Resources
 --------------------
 
+A description of the HYDRAD code is available on its Github repository,
+
+-  `HYDRAD user's guide
+   <https://github.com/rice-solar-physics/HYDRAD/blob/master/HYDRAD_User_Guide(03_20_2021).pdf>`__
```

Comment:
```suggestion
   <https://github.com/rice-solar-physics/HYDRAD/blob/6344b8e3e14ba7d3d470f9a5d57b0adc16421731/HYDRAD_User_Guide(03_20_2021).pdf>`__
```
Linking to the commit hash to avoid broken links in the future.